_posts/2023-03-22-dd_00005_half_adder.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: single
+title: Combinatorial Circuits - The Half Adder
+category: Digital Design
+permalink: /categories/digital_design/half_adder
+excerpt: "How to add 1-bit numbers in hardware."
+description:
+  "This post introduces the Half-Adder, a circuit which forms the basis of any
+  digital addition circuitry."
+author: Kirthihan Yasotharan
+toc: false
+toc_label: "Content"
+toc_sticky: true
+toc_icon: "align-justify"
+header:
+  overlay_color: "#000"
+  overlay_filter: "0.5"
+  overlay_image: /assets/images/digital_design.jpg
+---
+
+In our last two posts we have learned what logic gates are, which kind of gates
+exist and how they can be implemented using transistors.
+
+Now it's time to see what all stuff we can do with these things! Have you ever
+wondered why a computer is called "computer"? Well, as the name suggests,
+computers calculate, or _compute_, stuff. What and how they do it is something
```

Comment:
maybe not again the word "stuff"

Reply:
Fixed: [9c71535](https://github.com/bitsandvolts/bitsandvolts.github.io/pull/49/commits/9c71535a9711473a1972f814625dde3ef1fc7af2)

Is "things" better?

Reply:
see my suggestions below